src/main/scala/it/pagopa/interop/apigateway/server/impl/Dependencies.scala

```diff
@@ -51,13 +50,7 @@ trait Dependencies {
 
   val rateLimiterDirective: ExecutionContext => Seq[(String, String)] => Directive1[Seq[(String, String)]] = {
     val logger: LoggerTakingImplicit[ContextFieldsToLog] = Logger.takingImplicit[ContextFieldsToLog](this.getClass)
-    ec =>
-      contexts => {
-        RateLimiterDirective.rateLimiterDirective(
-          rateLimiter,
-          problemOf(StatusCodes.TooManyRequests, GenericComponentErrors.TooManyRequests)
-        )(contexts)(ec, entityMarshallerProblem, logger)
-      }
+    ec => contexts => RateLimiterDirective.rateLimiterDirective(rateLimiter)(contexts)(ec, serviceCode, logger)
```

Comment:
rate limiting now uses `Problem` in commons.
See [here](https://github.com/pagopa/interop-commons/pull/243/commits/1a683d4a97e37694b9a8c81d4aeeaa78aad3d5b7)